### PR TITLE
Feat/choosing format for saving files

### PR DESCRIPTION
**Pull request by zabukovnikmaj**

### CODE REVIEW

File: Models/BaseModel.php [hunk 1]
```diff
@@ -3,19 +3,30 @@
 namespace Models;
 
 use Exception;
+use SimpleXMLElement;
```
Comment (vojkovoga) [inline Models/BaseModel.php:6]: Do you need this import?

File: Services/Storage.php [hunk 2]
```diff
@@ -41,25 +12,63 @@ public function addElement(string $tableName, Model $model): void
...
+     */
+    public static function changeDataFilesToCorrectFormat(): void
+    {
+        $tableNames = ['Products', 'Employees', 'BranchOffice'];
+        $tableContents = [];
+
+        foreach ($tableNames as $tableName) {
+            foreach (CONFIG['possibleStorageMethods'] as $extension) {
+                $dir = storage_path($tableName . '.' . $extension);
+                if (file_exists($dir)) {
```
Comment (vojkovoga) [inline Services/Storage.php:54]: You can combine those two `if` statements together

**Review by vojkovoga — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,4 @@
 /.idea
 /data/*
-/*.jpg
\ No newline at end of file
+/*.jpg
+/config/config.php
```
[the remark this answers is not in the document]
Comment (zabukovnikmaj) [reply .gitignore:4]: fixed

File: Models/BaseModel.php (hunk 1 above)
Comment (zabukovnikmaj) [reply Models/BaseModel.php:6]: unnecessary import has been removed

File: Services/Storage.php (hunk 2 above)
Comment (zabukovnikmaj) [reply Services/Storage.php]: combined

**Pull request merged**